reconcile/jira_permissions_validator.py

```diff
@@ -243,8 +242,9 @@ def validate_boards(
                     # no errors
                     logging.debug(f"[{board.name}] is valid")
                     if not dry_run:
-                        # remember time of the last successful run
-                        state[board.name] = time.time()
+                        # remember time of the last successful run and add a random offset
+                        # to avoid all boards being checked at the same time
+                        state[board.name] = time.time() + random.randint(0, 3600)
```

Comment:
how about make the value to be next run time (`current_time + fixed_interval + random_jit)`, so to determine run or not, just compare current time with saved state time.

Reply:
You're right; it's easier to understand. Done in d2e6a0e.